R/FilterStateDate.R

```diff
@@ -109,7 +109,7 @@ DateFilterState <- R6::R6Class( # nolint
     #'   if `extract_type` argument is not empty.
     #' @param varname (`character(1)`)\cr
     #'   name of the variable.
-    #' @param choices (`atomic`, `NULL`)\cr
+    #' @param choices (`vector`, c(min(x, na.rm = TRUE), max(x, na.rm = TRUE)))\cr
```

Comment:
I don't think I understand this specification here. The help should only list allowed types of input, not values, should it? If this is to be a default value, it should be put in the definition, not the help. Also, `range`.

```suggestion
    #' @param choices (`vector`, `range(x, na.rm = TRUE)`)\cr
```